clients/admin-ui/src/features/privacy-requests/configuration/StorageConfiguration.tsx

```diff
@@ -0,0 +1,133 @@
+import Layout from "~/features/common/Layout";
+import {
+  useGetStorageDetailsQuery,
+  useSetActiveStorageMutation,
+} from "~/features/privacy-requests/privacy-requests.slice";
+
+import S3StorageConfiguration from "./S3StorageConfiguration";
+
+interface StorageData {
+  type: string;
+  details: {
+    auth_method: string;
+    bucket: string;
+  };
+  format: string;
+}
+
+const StorageConfiguration = () => {
+  const [existingStorageData, setExistingStorageData] = useState<StorageData>();
+  const { data: existingData } = useGetStorageDetailsQuery(
+    existingStorageData?.type
+  );
+  const [isLoading, setIsLoading] = useState(false);
+  const { errorAlert, successAlert } = useAlert();
+  const [saveStorageType] = useSetActiveStorageMutation();
+
+  const handleChange = async (value: string) => {
+    setIsLoading(true);
+
+    const payload = await saveStorageType({
+      active_default_storage_type: value,
+    });
+    if ("error" in payload) {
+      errorAlert(
+        getErrorMessage(payload.error),
+        `Configuring storage type has failed to save due to the following:`
+      );
+      setIsLoading(false);
+    } else {
+      successAlert(`Configure storage type saved successfully.`);
+      setExistingStorageData(existingData);
+      setIsLoading(false);
+    }
+  };
+
+  return (
+    <Layout title="Configure Privacy Requests - Storage">
+      <Box mb={8}>
+        <Breadcrumb fontWeight="medium" fontSize="sm" color="gray.600">
+          <BreadcrumbItem>
+            <BreadcrumbLink as={NextLink} href="/privacy-requests">
+              Privacy requests
+            </BreadcrumbLink>
+          </BreadcrumbItem>
+          <BreadcrumbItem>
+            <BreadcrumbLink as={NextLink} href="/privacy-requests/configure">
+              Configuration
+            </BreadcrumbLink>
+          </BreadcrumbItem>
+          <BreadcrumbItem color="complimentary.500">
+            <BreadcrumbLink
+              as={NextLink}
+              href="/privacy-requests/configure/storage"
+              isCurrentPage
+            >
+              Configure storage
+            </BreadcrumbLink>
+          </BreadcrumbItem>
+        </Breadcrumb>
+      </Box>
+
+      <Heading mb={5} fontSize="2xl" fontWeight="semibold">
+        Configure storage
+      </Heading>
+
+      <Box display="flex" flexDirection="column" width="50%">
+        <Box>
+          To configure a Storage destination, first choose a method to store
+          your results. Fides currently supports{" "}
+          <Text as="span" color="complimentary.500">
+            Local
+          </Text>{" "}
+          and{" "}
+          <Text as="span" color="complimentary.500">
+            S3
+          </Text>{" "}
+          storage methods.
+        </Box>
+        <Heading fontSize="md" fontWeight="semibold" mt={10}>
+          Choose storage type to configure
+        </Heading>
+        <RadioGroup
+          _disabled={isLoading}
```

Comment:
Chakra [has `isDisabled` ](https://chakra-ui.com/docs/hooks/use-radio-group#:~:text=the%20radio%20group.-,isDisabled,-boolean)